### DISCUSSION TITLE
Running in Proxmox

### DISCUSSION THREAD

**Discussion by ciband** [Start9Labs/start-os#415] [Q&A] [answered]
Hello,

I love the idea of what Start9 is doing.  

Is it possible to compile embassy-os for a non-ARM platform to make it easy to run in a VM under something like Proxmox?  Alternatively, do you know if anyone has had success running embassy-os in Proxmox using some sort of ARM emulator?

Thank you.

**Comment by kn0wmad** [comment]
Hi there, thanks for reaching out!

Currently only ARM builds are supported, although we will expand to more architectures.  It is possible to emulate ARM in a VM, but we do not support that.  You might find it helpful to join our Community Dev Matrix channel, where others are interested (and may be experimenting) in these objectives.  

Please let me know if you have any further questions :)

**Reply by ProofOfKeags** [reply]
To add to this, the main issue isn't ARM vs x86, but rather that we do make various assumptions for how the system is laid out on the OS side. If you are deploying to a Ubuntu for RPi VM you could deploy embassy-os on its own. The real problem is that all of the packages themselves are built *only* for ARMv7 in 0.2.x and *only* for ARMv8 in 0.3.x. There are future plans for multi-arch packages but as of today they are thoroughly unsupported.

**Comment by kn0wmad** [comment, marked as answer]
Hi there @ciband - just wanted to let you know that you can now run eOS (as of today's release of v0.3.3) on Promox :)

Good luck on your sovereign journey!

**Reply by ciband** [reply]
@kn0wmad , Sweet!  Thank you for the update.
